### DISCUSSION TITLE
an error when clicking on myMastodonserver/about/more

### DISCUSSION THREAD

**Discussion by vserbu** [mastodon/mastodon#17212] [Q&A] [answered]
Hi,

I'm new Mastodon administrator but I have a problem.  I started to customize my instance and tried to write down some of instance rules via administration -> site settings -> Custom extended information and everything worked well. But now when I click on mymastodonserver/about/more (server rules) i get " We're sorry, but something went wrong on our end. " . I even tried to delete everything from that field ("Custom extended information") but there is no change. I restarted the server, mastodon. What can I do? 

NEW: 
Ok, I narrowed it down, it is problem with localization. I entered a line in .env.production to make it defalut locale DEFAULT_LOCALE=hr ant then it won't open the about/more page. When I removed it, it is working (but in English). 

It is definetly connected with localization of that page. When I open Mastodon index page in browser with default Croatian language, I get the same problem. If I open the same index page in browser with default English language, everything is ok.

**Comment by ClearlyClaire** [comment, marked as answer]
That sounds like a bug in Mastodon. Please open a new issue! It could also be helpful if you could provide the error trace from `journalctl -u mastodon-web`, but in this specific case, I think you gave enough information to reproduce and identifiy the problem.

**Reply by vserbu** [reply]
I will, tnx!
